apps/google-analytics-4/tsconfig.json

```diff
@@ -0,0 +1,14 @@
+{
+  "extends": "@tsconfig/create-react-app/tsconfig.json",
+  "compilerOptions": {
+    "target": "es5",
+    "lib": [
+      "dom",
+      "dom.iterable",
+      "esnext"
+    ],
```

Comment:
Probably we can remove those from the template... not sure why we still have them in there.